docs/UsersGuide.asciidoc

```diff
@@ -364,7 +364,8 @@ Based on comments on the group overview individual builds can be tagged. As
 'build' by themselves do not own any data the job group is used to store this
 information. A tag has a build to link it to a build. It also has a type
 and an optional description. The type can later on be used to distinguish
-tag types.
+tag types.  Note that openQA does not define further tag types besides the _important_ tag.
```

Comment:
```suggestion
tag types. Note that openQA does not define further tag types besides the _important_ tag.
```

Reply:
and please also wrap lines consistently at 80 characters

Reply:
*this change is ready for review*